NitroxClient/Communication/MultiplayerSession/ConnectionState/Disconnected.cs

```diff
@@ -66,7 +66,8 @@ private static void StartClient(string ipAddress, IClient client)
 
         private static void EstablishSessionPolicy(IMultiplayerSessionConnectionContext sessionConnectionContext, IClient client)
         {
-            string policyRequestCorrelationId = Guid.NewGuid().ToString();
+            ulong steamId = Steamworks.SteamUser.GetSteamID().m_SteamID;
```

Comment:
Would this throw an exception if steam was not authenticated? (duplicate instances for testing)